lang/python/python-mako/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-mako
+PKG_VERSION:=1.1.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=Mako-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/m/mako/
```

Comment:
We can use PyPI variables and such, right?
https://github.com/openwrt/packages/blob/master/lang/python/README.md#include-pypimk-optional

Reply:
yes, thanks for the hint